iOS/ViewControllers/Courses/CourseItemViewController.swift

```diff
@@ -137,3 +184,13 @@ extension CourseItemViewController: UIPageViewControllerDelegate {
     }
 
 }
+
+extension CourseItem: CourseItemURL {
+
+    public var courseItemURL: URL? {
+        guard let courseId = self.section?.course?.id else { return nil }
+        guard let courseItemId = self.base62id else { return nil }
+        return Routes.courses.appendingPathComponents([courseId, "items", courseItemId])
```

Comment:
To be consistent to the web, you can use the course slud instead of the course id.